pennylane/transforms/condition.py

```diff
@@ -111,14 +111,8 @@ def qnode(x, y):
 
     .. warning::
 
-        The following are not supported as the ``condition`` argument:
-
-        * Expressions that contain multiple measurement values;

```

Comment:
so we support this now?

Reply:
`m0 & m1` for example is a valid condition. I assume this was an old comment that got forgotten.

Reply:
right, I think we talked about this with Tom.